Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -336,16 +336,19 @@ public IEnumerator ManipulationHandlerOneHandMoveNear()
                 yield return hand.MoveTo(initialGrabPosition, numHandSteps);
```

Comment:
It would be very helpful to add a test case for move/scale/rotate object when center != mesh center. I could see this as a feature that somebody could accidentally break in the future, we should protect against that since it was so hard to get working.